package.json

```diff
@@ -70,6 +70,11 @@
     "react-native-builder-bob": "^0.18.0",
     "typescript": "^4.1.3"
   },
+  "dependencies": {
+      "@segment/analytics-react-native": "https://gitpkg.now.sh/customerio/analytics-react-native/packages/core?aman/fork-integrate-11653",
+      "@segment/sovran-react-native": "https://gitpkg.now.sh/customerio/analytics-react-native/packages/sovran?aman/fork-integrate-11653",
+      "react-native-get-random-values": "^1.9.0"
+    },
```

Comment:
The changes in `package.json` looks good. I don't see a `yarn.lock` file change in this PR though? I was expecting the yarn lock to update since the npm dependencies have changed.

Reply:
The CI errors may also be fixed by updating the yarn lock file it looks like.